eng/SourceBuild.props

```diff
@@ -8,6 +8,8 @@
   <Target Name="ConfigureInnerBuildArgs" BeforeTargets="GetSourceBuildCommandConfiguration">
     <PropertyGroup>
       <InnerBuildArgs>$(InnerBuildArgs) /p:Projects="$(InnerSourceBuildRepoRoot)\MSBuild.SourceBuild.slnf"</InnerBuildArgs>
+      <!-- Disable package validation as source build filters out target frameworks. -->
+      <InnerBuildArgs>$(InnerBuildArgs) /p:EnablePackageValidation=false</InnerBuildArgs>
```

Comment:
Is this better than putting a condition on the enabling property definitions? It "feels" more invasive to me but I wouldn't block on this.

Reply:
We do the same in runtime: https://github.com/dotnet/runtime/blob/e395c7c767437b9b8d10113bb8fc816ffb30f521/eng/SourceBuild.props#L51.

We don't want package validation to run at all during source build as in that mode we filter TFMs out that aren't source buildable and that would change the emitted APICompat suppressions. I don't think we want extra suppression files just for the source build mode.